### PR TITLE
Fix Comunity, Foundation and software vendor pages

### PR DESCRIPTION
**Pull request by MatiasOS**
### Components
- About
- Collaborate

### Improvements
- Add how we work to banners
- Update Card component

### CODE REVIEW

File: src/features/Foundation/About/index.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            Winding Tree Foundation is a non-profit organization incorporated in Switzerland.
+             Its purpose is to develop software projects, data exchange standards and
+            infrastructure with the goal of advancing the travel industry.
+          </p>
+          <p className="lead mb-1 mb-md-2 text-center">
+            Winding Tree foundation currently supports these initiatives:
+          </p>
+          <CardDeck>
+            {cardsData.map(card => (
+              <Card {...card} key={`about-card-${card.title}`} />
```
Comment (PabloAlice) [inline src/features/Foundation/About/index.jsx]: it's not bad.
just FYI, the key attribute is not needed for the browser or assistance stuff. it's just for react, and the only constraint is being unique. so `'about-card-{card.title}'` or `card.title` works. Theoretically, about-card is totally redundant.

File: src/features/Foundation/index.jsx [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 import React from 'react';
 import Banner from 'commons/Banner';
+import Events from 'commons/Events';
 import Hero from 'commons/Hero';
 import Team from './Team';
+import About from './About';
 import { members, advisors } from './data';
 
 const Foundation = () => (
   <main className="app-content">
```
Comment (skkr) [inline src/features/Foundation/index.jsx]: this should be an id

Comment (MatiasOS) [reply src/features/Foundation/index.jsx]: Done

**Comment by menschikov** [comment]
@MatiasOS seems we lost a space 
image

Seem this space run away to community page bottom :D
image

**Pull request merged**